### PR TITLE
Update UG for find by email and address

### LINKED ISSUE #207: Update UG to include find by email, address

### PR DESCRIPTION
**Pull request by gnahzeus**
Closes #207

### CODE REVIEW

**Comment by seanflyyy** [comment]
LGTM!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -415,6 +415,41 @@ Example:
 
 ![UiFindName](images/UG-screenshots/UiFindName.png)
 
+#### Find by Email:
+
+Finds all students with a particular email.
+
+Formats: `find e/EMAIL
```
Comment (seanflyyy) [inline docs/UserGuide.md:422]: Should be `find e/EMAIL` 

Missing a backtick

Comment (seanflyyy) [reply docs/UserGuide.md:422]: Also, I think it should be format not formats

**Review by seanflyyy — CHANGES_REQUESTED** [review]
Minor formatting issues

**Review by Echomo-Xinyu — APPROVED** [review]
LGTM except for the minor format error as mentioned by @seanflyyy !

File: docs/UserGuide.md [hunk 2]
```diff
@@ -415,6 +415,41 @@ Example:
...
+- Only one email can be searched at each time.
+- Full email must be used for corresponding student to be found e.g., `ghost` will not match `ghost@woods.com`.
+
+```yaml
+❗Caution:
+  Do not include more than one email such as
+  find e/jonsnow@winterfell.com ghost@woods.com.
+```
+
+Examples:
```
Comment (sjoann) [inline docs/UserGuide.md:434]: I think this should be Example: 
or maybe can add more examples below

File: docs/UserGuide.md [hunk 3]
```diff
@@ -415,6 +415,41 @@ Example:
...
+
+`find e/jonsnow@winterfell.com` returns all students with email set as `jonsnow@winterfell.com`.
+
+#### Find by address:
```
Comment (sjoann) [inline docs/UserGuide.md:438]: maybe can capitalise Address to standardise

File: docs/UserGuide.md [hunk 4]
```diff
@@ -415,6 +415,41 @@ Example:
...
+
+Finds all students with addresses matching the keywords.
+
+Format: `find a/KEYWORD [MORE_KEYWORDS]`
+
+- The search is case-insensitive. e.g., `Labrador` will match `LABRADOR`.
+- The order of the keywords does not matter. e.g. `Bukit Timah` will match `Timah Bukit`.
```
Comment (sjoann) [inline docs/UserGuide.md:445]: can add a comma e.g. --> e.g.,

File: docs/UserGuide.md [hunk 5]
```diff
@@ -415,6 +415,41 @@ Example:
 
 ![UiFindName](images/UG-screenshots/UiFindName.png)
 
+#### Find by Email:
+
+Finds all students with a particular email.
+
+Formats: `find e/EMAIL
+
+- The search is case-insensitive. e.g., `ghost@woods.com` will match `ghOst@woOds.com`.
```
Comment (sjoann) [inline docs/UserGuide.md:424]: this looks like zero 0 instead of capital O letter but I am not sure if this is my laptop problem

**Review by sjoann — APPROVED** [review]

**Review by seanflyyy — APPROVED** [review]

**Pull request merged**